### DISCUSSION TITLE
Tilgangen til APIet på AKS bør begrenses til gitte IP-adresser

### DISCUSSION THREAD

**Discussion by annerisbakk** [Altinn/altinn-platform#2380] [Ideas]
https://dev.azure.com/digdir/Altinn/_workitems/edit/43753
Ref: https://docs.microsoft.com/en-us/azure/aks/api-server-authorized-ip-ranges

Her må en vurdere hvordan dette passer med Azure DevOps og terraform.

- [ ] Begrense tilgangen til platform-kluster APIet

- [ ] Begrense tilgangen til Altinn Studio APIet

- [ ] Begrense tilgangen på AKS APIet til tjenesteeierkluster

- [ ] Vurdere om vi kan begrense IP-adressene tilstrekkelig

**Comment by SandGrainOne** [comment]
We've added a dependency to issue #231. We would like a mechanism were we can avoid the need for a an extra virutal machine just to access resources in Azure.

**Comment by ghost** [comment]
@SandGrainOne [This](https://docs.microsoft.com/en-us/azure/vpn-gateway/work-remotely-support) might be a mechanism to consider.

**Comment by bengtfredh** [comment]
Setting restriction on AKS api will prevent use of virtual nodes:

https://learn.microsoft.com/en-us/azure/aks/virtual-nodes#:~:text=Using%20api%20server%20authorized%20ip%20ranges%20for%20AKS.
